### ISSUE TITLE
while executing this line - !pip install --upgrade git+https://github.com/ramsrigouthamg/Questgen.ai

### ISSUE THREAD

**Issue by kLolayekar** [ramsrigouthamg/Questgen.ai#56]
Using cached sentencepiece-0.1.97-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.3 MB)
Requirement already satisfied: filelock in /usr/local/lib/python3.9/dist-packages (from transformers==3.0.2->Questgen==1.0.0) (3.9.0)
Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.9/dist-packages (from transformers==3.0.2->Questgen==1.0.0) (2022.6.2)
Collecting tokenizers==0.8.1.rc1
  Using cached tokenizers-0.8.1rc1.tar.gz (97 kB)
  Installing build dependencies ... done
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Getting requirements to build wheel ... error
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

**Comment by Aavish-Gilbert-J** [comment]
I have the same error! Did you sort it out?

**Comment by kLolayekar** [comment]
Yes, it worked for me with Python version 3.8

**Comment by respectmashego** [comment]
KLolayekar did you able  to install all the libraries?

**Comment by Aavish-Gilbert-J** [comment]
> KLolayekar did you able to install all the libraries?

No, It didn't work for me with pip and python 3.8. I had to manually install the library from github with python 3.9 and it worked.

**Comment by respectmashego** [comment]
i am struggiling this side.did you able to run those code successfully,like experimenting the library?

**Comment by Aavish-Gilbert-J** [comment]
Yes, it was successful... The source code for the library is available, i had to manually clone it and for some reason it worked only for python 3.9.

**Comment by respectmashego** [comment]
How did you get the source code?
Im struggling at installation 1.2 at sense2vec  please  help me.

**Comment by Dp95** [comment]
> Yes, it worked for me with Python version 3.8

how did it work

**Comment by Imzachjohnson** [comment]
Be sure to use a python version in the range of 3.10.x or below. The numpy and pandas requirements conflict at a higher version.

**Comment by AliHaider20** [comment]
This issue is solved. Close it.
